README.md

```diff
@@ -12,28 +12,35 @@ HPE OneView 4.0 version or above is required.
 
 > In order to integrate properly with OneView, the OneView API 600 is required to be supported by OneView instance.
 
-## Installation
+# Getting started
 
-### Requirements
+## Requirements
 
-HPE OneView Redfish Toolkit service relies on Python 3.5 or newer (as long as python3 executable is available) to run and [pip3](https://pip.pypa.io/en/stable/installing/) for dependencies management. A full list of dependencies is available at [requirements.txt](requirements.txt) file. For pyOpenSSL module please make sure to have OpenSSL lib installed in your system.
+* Ubuntu 16.04
```

Comment:
Is Ubuntu 16 what version everyone tested with?
Should openssl be on the requirements list?  A specific version?
Any other requirements here?